app/app/settings.py

```diff
@@ -159,8 +159,7 @@
     'corsheaders.middleware.CorsMiddleware',
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
-    'app.middleware.drop_accept_langauge',
-    # 'app.middleware.bleach_requests',
```

Comment:
This seems to have been commented out 2 years ago, so I think it is safe to assume we will not enable it back.